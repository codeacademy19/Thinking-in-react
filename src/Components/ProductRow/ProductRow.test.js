import React from 'react';
import renderer from 'react-test-renderer';
import ProductRow from '.';
import {shallow} from 'enzyme';


describe('ProductRow',()=>{
    const productStocked ={category: 'Sporting Goods', price: '$49.99', stocked: true, name: 'Football'};
    const productNotStocked ={category: 'Sporting Goods', price: '$49.99', stocked: false, name: 'Football'};
        // {category: 'Sporting Goods', price: '$9.99', stocked: true, name: 'Baseball'},
        // {category: 'Sporting Goods', price: '$29.99', stocked: false, name: 'Basketball'},
        // {category: 'Electronics', price: '$99.99', stocked: true, name: 'iPod Touch'},
        // {category: 'Electronics', price: '$399.99', stocked: false, name: 'iPhone 5'},
        // {category: 'Electronics', price: '$199.99', stocked: true, name: 'Nexus 7'};
    it('should render a row with value as Product Category',()=>{
        const tree=renderer.create(<ProductRow  product={productStocked}/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it('should render a row with value as Product Category',()=>{
        const tree=renderer.create(<ProductRow  product={productNotStocked}/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
});