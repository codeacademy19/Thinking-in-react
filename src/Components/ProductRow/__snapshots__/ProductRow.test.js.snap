// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductRow should render a row with value as Product Category 1`] = `
<tr>
  <td>
    Football
  </td>
  <td>
    $49.99
  </td>
</tr>
`;

exports[`ProductRow should render a row with value as Product Category 2`] = `
<tr>
  <td>
    <span
      style={
        Object {
          "color": "red",
        }
      }
    >
      Football
    </span>
  </td>
  <td>
    $49.99
  </td>
</tr>
`;
